### PR TITLE
Publish to Maven Central

### PR DESCRIPTION
**Pull request by aaukhatov**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -16,10 +20,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 15
+    - name: Set up JDK 8
```
Comment (ilyavy) [inline .github/workflows/gradle.yml]: Is it made to allow usage of the library with java 8+?

**Review by ilyavy — APPROVED** [review]
Cool!

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (aaukhatov) [reply .github/workflows/gradle.yml]: Yes, it is

Comment (aaukhatov) [reply .github/workflows/gradle.yml]: I decided to expand our potential users.
I'm assuming users of Java 8 more than users of Java 8+

**Pull request merged**